server/app/middlewares/websocket_backend.rb

```diff
@@ -172,30 +173,28 @@ def find_node(req)
   # @param [Faye::WebSocket] ws
   # @param [Rack::Request] req
   def on_open(ws, req)
+    node_id = req.env['HTTP_KONTENA_NODE_ID']
+    agent_version = req.env['HTTP_KONTENA_VERSION'].to_s
+    connected_at = req.env['HTTP_KONTENA_CONNECTED_AT']
+    connected_at = Time.parse(connected_at) if connected_at
```

Comment:
Yes, because it's now used in the `rescue CloseError` that can be raised by the agent version check, and that can be triggered by pre-1.2 agents... changed this to just explicitly initialize it to nil.